.snyk

```diff
@@ -0,0 +1,10 @@
+# Snyk (https://snyk.io) policy file, patches or ignores known vulnerabilities.
+version: v1.14.1
+ignore:
+  SNYK-JS-NODENOTIFIER-1035794:
+    - '* > node-notifier':
```

Comment:
It's not clear from the ticket why node-notifier is included in this file. Can you provide some background?

Reply:
Sure, the Command Injection ID for this issue [references node-notifier](https://snyk.io/vuln/SNYK-JS-NODENOTIFIER-1035794). Snyk doesn't appear to be upgrading node-notifier in our case but I included it in case this was part of the issue.